README.md

```diff
@@ -15,3 +15,74 @@ Accessibility
 
 Evaluation Date: 201X-XX-XX  
 This project conforms to level AA WCAG 2.0 standards as required by the university's accessibility policy. For more information on the policy please visit: [https://ucomm.stanford.edu/policies/accessibility-policy.html](https://ucomm.stanford.edu/policies/accessibility-policy.html).
+
+System Requirements
+---
+
+To build and deploy code, you must have the following installed on your system:
+
+* Node
+* Grunt
+
+
+### Installation
+
+These tools need only to be installed globally, so if you already have them installed on your system, you are ready to use or add to the toolset.
+
+You may refer to the respective links above for installation instructions for each, but below is a brief overview for each as well:
+
+### Node
+
+Visit Node's website to download the [installer](http://nodejs.org/).
+
+### Grunt
+
+The only requirement for Grunt is Grunts CLI(Command Line Utility).
+If you don't already have that installed, install globally, this is not project specific.
+In NPM we do so with the -g flag.
+
+    npm install -g grunt-cli
+
+### Troubleshooting
+
+* If you have trouble installing any of the tools, you may need to use the sudo command.
+* If you run into any issues with some of the tasks run, npm rebuild and try again.
+`npm rebuild`
+* Also make sure that git is installed globally as some bower packages require it to be fetched and installed.
+
+Development Workflow
+---
+
+### One-time Setup
+
+The following steps need to be performed once to get your local environment up and running.
```

Comment:
Lando is optional and not a requirement. I do not want to force anyone to use this webserver if they don't want to. Perhaps we could change this section to an optional install and move the KSS setup to the Installation section?